core/src/main/java/novoda/lib/sqliteprovider/util/UriUtils.java

```diff
@@ -6,29 +6,28 @@
 
 public class UriUtils {
 
-    private static Map<String, String> mappedIds = new HashMap<String, String>();
+    private UriUtils() throws IllegalAccessException {
+        throw new IllegalAccessException("Utils class shouldn't be instantiated");
```

Comment:
Why throwing? It's also the wrong exception:

>An IllegalAccessException is thrown when an application tries to reflectively create an instance (other than an array), set or get a field, or invoke a method, but the currently executing method does not have access to the definition of the specified class, field, method or constructor.

This is not reflective. It should throw `InvalidOperationException` or something similar, if anything, but I'd simply do nothing in there.